packages/cxl-lumo-styles/scss/_mixins.scss

```diff
@@ -77,13 +76,41 @@
   background-position: bottom left calc(50vw + clamp(48px,5%,100px)) !important;
 
   &::before {
-    left: 47%;
-    height: 60%;
+    left: 50%;
     border-top-right-radius: 0;
     border-bottom-right-radius: 0;
   }
 }
 
+@mixin wide-background-hero {
+  &::before {
+    height: clamp(164px, calc(2 * var(--figure-height-factor)), calc(100 * var(--figure-height-factor))/2);
+    bottom: 50%;
+    transform: translateY(50%);
+  }
+}
+
+// Use ::after pseudo element to add hero background image. url must be set to CSS custom property --hero-image
+@mixin wide-background-hero-image {
+  &::after {
+    content: "";
+    position: absolute;
+    left: 55%;
+    width: 100%;
+    height: clamp(calc(50% + 82px), calc(50% + var(--figure-height-factor)), calc(50% + var(--figure-height-factor)));
+    max-height: calc(50% + var(--figure-height-factor));
+    background-image: var(--hero-image);
```

Comment:
How does this line correspond with

```css
#view-hero::after {
    background-image: url([wpv-post-featured-image size="xx-large" output="url"]);
}
```

Reply:
I just spotted PR comment has slightly different instructions:

```
style="--hero-image: url([wpv-post-featured-image size="large" output="url"])"
```

@freudFlintstone please update component description: https://github.com/conversionxl/aybolit/pull/365/commits/67df340df25f96a293c0bdde7a100a973413eae5#diff-b081320cd271d2ea1629b6ca196f8dcc1df161b415770b0cdb98e5b53f1f15ffR19